api/client/stack/config.go

```diff
@@ -33,7 +33,7 @@ func newConfigCommand(dockerCli *client.DockerCli) *cobra.Command {
 }
 
 func runConfig(dockerCli *client.DockerCli, opts configOptions) error {
-	bundle, err := loadBundlefile(dockerCli.Err(), opts.namespace, opts.bundlefile)
+	bundle, err := loadBundlefile(dockerCli.Out(), opts.namespace, opts.bundlefile)
```

Comment:
I think `stderr` is correct here. This output is logging, not "normal program output".